kernel/console/kprintf.c

```diff
@@ -73,11 +72,11 @@ uint64_t kprintf_do_uint(uint8_t width, char *output, uint8_t base, __builtin_va
 		case 16:
 		case 32:
 			conv_uint = __builtin_va_arg(ap, unsigned int);
-			uitoa3(conv_uint, output, 66, base);
+			uitoa3(conv_uint, output, KPRINTF_LEN, base);
```

Comment:
I can't imagine that there'd be any reason for the # of chars in the output string to be over 66 in `kprintf_do_uint()`.  This function is concerned specifically with stringifying unsigned integer values, so if we do a conversion to a binary representation, then at most we'll have 64 bytes to display the stringified value (for a uint64_t), an optional "b" sigil at the end, and \0.